### ISSUE TITLE
Org admin cannot archive own project

### ISSUE THREAD

**Issue by seav** [Cadasta/cadasta-platform#1752]
### Steps to reproduce the error
I don't know how to reproduce this, but this is happening on staging. I think this was a side-effect of the tutelary replacement branch testing. I am an org administrator for a project (see 1st screenshot below), but when I try to archive a project, I get a denied error message (see 2nd screenshot below).

![screen shot 2017-08-29 at 23 46 55](https://user-images.githubusercontent.com/873653/29830385-ae9754d0-8d14-11e7-8e98-fa72f94aa9f1.png)

![screen shot 2017-08-29 at 23 47 08](https://user-images.githubusercontent.com/873653/29830395-b457a4ce-8d14-11e7-913b-a933d543628d.png)

### Actual behavior
Org admin cannot archive a project in their org.

### Expected behavior
Org admin should be able to archive a project in their org.

**Comment by oliverroick** [comment]
Have you created that project before the permissions branch went on staging? I can archive projects in an org I have created today.

**Comment by seav** [comment]
I created that project today, but the organization and the admin account existed before the permissions branch went on staging.

**Comment by amplifi** [comment]
I can archive projects on orgs pre- and post- permissions branch. In your screenshot, the account is listed as admin but has different non-admin project roles. In viewing the organization's member list now, it shows you as a member with project manager roles:

<img width="1162" alt="screen shot 2017-08-29 at 5 12 01 pm" src="https://user-images.githubusercontent.com/4269749/29831477-359c3b90-8cdd-11e7-83e7-66f03d365eb2.png">

It looks like somewhere in testing with your organization, user permissions got modified, but this is unrelated to the permissions branch.

**Comment by seav** [comment]
OK. So there is definitely some weirdness going on and a most likely an inconsistency in the database. Only an org admin can create projects and view the org membership page (otherwise, I wouldn't be able to take that screenshot) or create the "QA Day Test Project", which I was trying to archive. So my account is being treated as an org admin in some respects but as a non-org admin in other respects.

**Comment by dpalomino** [comment]
Thanks @seav. Definitely it seems related with #1754. In my case, if you create a new organization then behaviour was fine, but with previously existing orgs it was failing.

**Comment by amplifi** [comment]
Ref: https://github.com/Cadasta/cadasta-platform/issues/1754#issuecomment-326063258

**Comment by oliverroick** [comment]
@seav Can you walk me through the steps you took to get to this stage? I can reproduce David's bug but not yours.

**Comment by seav** [comment]
@oliverroick, I can't remember if I did anything to get this bug so I stated above that "I don't know how to reproduce this". I definitely did not attempt to remove my account's administrator status during testing. But if your explanation in #1754 is correct, then this bug seems to make sense. Only org admins can archive/unarchive projects and while I was able to create a project (which is probably related to my existing tutelary `project.create` permission tied to the organization), my tutelary permissions denied me the permission to archive that project because my org role is that of a member instead of an admin and the tutelary `project.archive` permission is tied to the newly-created project and not to the preexisting org.
